### ISSUE TITLE
External access to XMPP actually not available?

### ISSUE THREAD

**Issue by poVoq** [ONLYOFFICE/Docker-CommunityServer#51]
While Talk works fine within the browser, I can't seem to get the external port 5222 to accept any XMPP client connections.

So far I though this is a problem with my docker setup, but then I read (I think) on of the OO staff saying this is only available in the enterprise edition?

Is that true and if so, could you please mention it in the Docker readme?

Thanks!

**Comment by Carazyda** [comment]
You can configure external ports through "docker run" command, by "-p" argument. We describe this in [exposing-additional-ports section](https://github.com/ONLYOFFICE/Docker-CommunityServer#exposing-additional-ports).

You deal whit this problem? Can connect external XMPP clients?

And where you seen that? We will correct this.

> only available in the enterprise edition

**Comment by poVoq** [comment]
Yes I am aware of that. What I did was tunnel the port through stunnel4 to get an SSL secured connection, but that really should not make a difference to the -p argument for a direct tunnel.

But if you confirm that the XMPP server is indeed not blocked in the community edition than I will investigate further and see if there is some issue with my stunnel setup.
